package.json

```diff
@@ -28,121 +28,121 @@
     }
   ],
   "dependencies": {
-    "@braid/vue-formulate": "^2.5.2",
+    "@braid/vue-formulate": "^2.5.3",
     "@chatwoot/prosemirror-schema": "https://github.com/chatwoot/prosemirror-schema.git",
     "@chatwoot/utils": "^0.0.16",
-    "@hcaptcha/vue-hcaptcha": "^0.3.2",
-    "@june-so/analytics-next": "^1.36.5",
-    "@rails/actioncable": "6.1.3",
-    "@rails/ujs": "^7.0.3-1",
+    "@hcaptcha/vue-hcaptcha": "^1.2.1",
+    "@june-so/analytics-next": "^2.0.0",
+    "@rails/actioncable": "7.0.6",
+    "@rails/ujs": "^7.0.6",
     "@rails/webpacker": "5.4.4",
     "@scmmishra/pico-search": "^0.5.1",
-    "@sentry/tracing": "^6.19.7",
-    "@sentry/vue": "^6.19.7",
-    "@sindresorhus/slugify": "1.1.0",
+    "@sentry/tracing": "^7.60.1",
+    "@sentry/vue": "^7.60.1",
+    "@sindresorhus/slugify": "2.2.1",
     "@tailwindcss/typography": "^0.5.9",
-    "activestorage": "^5.2.6",
+    "activestorage": "^5.2.8",
     "autoprefixer": "^10.4.14",
-    "axios": "^0.21.2",
+    "axios": "^1.4.0",
     "babel-plugin-syntax-jsx": "^6.18.0",
     "babel-plugin-transform-vue-jsx": "^3.7.0",
-    "chart.js": "~2.9.4",
-    "company-email-validator": "^1.0.8",
-    "core-js": "3.11.0",
-    "date-fns": "2.21.1",
-    "date-fns-tz": "^1.3.3",
-    "dompurify": "2.2.7",
-    "foundation-sites": "~6.5.3",
-    "highlight.js": "~10.4.1",
+    "chart.js": "~4.3.2",
+    "company-email-validator": "^1.0.9",
+    "core-js": "3.31.1",
+    "date-fns": "2.30.0",
+    "date-fns-tz": "^2.0.0",
+    "dompurify": "3.0.5",
+    "foundation-sites": "~6.7.5",
+    "highlight.js": "~11.8.0",
     "idb": "^7.1.1",
-    "ionicons": "~2.0.1",
-    "js-cookie": "^2.2.1",
-    "libphonenumber-js": "^1.10.24",
-    "logrocket": "^3.0.1",
+    "ionicons": "~7.1.2",
+    "js-cookie": "^3.0.5",
+    "libphonenumber-js": "^1.10.38",
+    "logrocket": "^4.0.4",
     "logrocket-vuex": "^0.0.3",
     "markdown-it": "^13.0.1",
     "markdown-it-link-attributes": "^4.0.1",
     "md5": "^2.3.0",
-    "ninja-keys": "^1.1.9",
+    "ninja-keys": "^1.2.2",
     "opus-recorder": "^8.0.5",
-    "pinia": "^2.1.4",
-    "postcss": "^8.4.24",
-    "postcss-loader": "^4.2.0",
-    "semver": "7.3.5",
-    "spinkit": "~1.2.5",
-    "tailwindcss": "^3.3.2",
+    "pinia": "^2.1.6",
+    "postcss": "^8.4.27",
+    "postcss-loader": "^7.3.3",
+    "semver": "7.5.4",
+    "spinkit": "~2.0.1",
+    "tailwindcss": "^3.3.3",
     "turbolinks": "^5.2.0",
-    "url-loader": "^2.0.0",
-    "urlpattern-polyfill": "^6.0.2",
+    "url-loader": "^4.1.1",
+    "urlpattern-polyfill": "^9.0.0",
     "v-tooltip": "~2.1.3",
-    "videojs-record": "^4.5.0",
-    "vue": "2.6.12",
-    "vue-chartjs": "3.5.1",
-    "vue-clickaway": "~2.1.0",
+    "videojs-record": "^4.6.0",
+    "vue": "3.3.4",
```

Comment:
This would completely break the web app, let alone wouldn't even compile.

Vue 3 introduces breaking changes.

Reply:
Will take a closer look